README.md

```diff
@@ -1,72 +1,106 @@
+## Apache Airflow 2 for local development and CI/CD
+=======================================================
+
+This project is an easy-to-use development environment for Apache Airflow version 2. It can be run locally on a variety of OS platforms with simple steps for spinning up Airflow. The project can be integrated into a CI/CD DATA pipeline using GCP Cloud Build to build, test, and deploy workflows into GCP Cloud Composer. The project is meant to address several "infrastructure challenges" that Airflow developers experience and allows them to focus on workflow development rather than platform installation/configuration.
```

Comment:
What's DATA pipeline?

Reply:
Can just have a bullet point list of features: container caching, unit tests, integration tests, linter, etc. (basically each step in cloud build lol )

Reply:
Also can say. "The project provides an opinionated CloudBuild CI/CD pipeline for Composer deployment" (Google likes the word opinionated lol )

Reply:
Could also divide it into 3 parts

1. Local Development environment
2. GCP cloud shell development environment 
3. Opinionated CI/CD pipeline

Reply:
- Changed DATA pipeline to CI/CD pipeline

Reply:
Revised the instructions in accordance with Eugene's feedback